feature/selector.py

```diff
@@ -13,7 +13,6 @@
 import numpy as np
 import pandas as pd
 import seaborn as sns
-from catboost import CatBoostClassifier, CatBoostRegressor
```

Comment:
Just for my own learning -- we are dropping CatBoost because it is ? slow? library version complaincy? 

Probbaly wanna mention this one in the Changelog

Reply:
Just considering it for now. Running into some discrepancies between running tests local vs using GH CI and the Catboost pip installation has been failing on and off. So I'm using GH to easily test across all OS and Python versions (and don't know how to do that without creating PR).

Feel free to ignore PR for now - I will tag you when ready.

Reply:
ah cool idea on using GH for testing 👍